src/test/java/org/apache/commons/text/similarity/JaccardDistanceTest.java

```diff
@@ -37,7 +37,7 @@ public static void setUp() {
     @Test
     public void testGettingJaccardDistance() {
         // Expected Jaccard distance = 1.0 - (intersect / union)
```

Comment:
It looks like this comment is out of sync with the code.

Reply:
Please, can you describe why this comment looks wrong? For me it's good, shows the formula, as well as in JaccardSimilarityTest.java
`// Expected Jaccard similarity = (intersect / union)`

Reply:
Because the comment reads:
```
// Expected Jaccard distance = 1.0 - (intersect / union)
```
and the next line is in contradiction with this comment, expecting 0.0, not 1.0.
```
assertEquals(0.0, classBeingTested.apply("", ""));
```

Reply:
Sorry @garydgregory , I'm still not convinced.
Intersection of two empty sets is empty set, union is also empty set. They are equal. So Jaccard similarity is 1. And therefore distance is 0.